src/modules/keepalive/keepalive_mod.c

```diff
@@ -52,19 +52,29 @@ static int ka_mod_add_destination(modparam_t type, void *val);
 int ka_init_rpc(void);
 int ka_alloc_destinations_list();
 extern void ka_check_timer(unsigned int ticks, void *param);
-
 static int w_cmd_is_alive(struct sip_msg *msg, char *str1, char *str2);
+static int fixup_add_destination(void** param, int param_no);
+static int w_add_destination(sip_msg_t *msg, char *uri, char *owner);
+static int w_del_destination(sip_msg_t *msg, char *uri, char *owner);
+
 
 extern struct tm_binds tmb;
 
 int ka_ping_interval = 30;
 ka_destinations_list_t *ka_destinations_list = NULL;
+str ka_ping_from = str_init("sip:keepalive@kamailio.org");
+int counter_del = 5;
```

Comment:
I suggest adding `ka_` prefix to global variables that are not static and exposed in other files via `extern`.